src/main/java/com/conveyal/datatools/manager/jobs/MergeStrategy.java

```diff
@@ -0,0 +1,46 @@
+package com.conveyal.datatools.manager.jobs;
+
+/**
+ * This enum defines the different strategies for merging, which is currently dependent on whether trip_ids and/or
+ * service_ids between the two feeds are exactly matching.
+ */
+public enum MergeStrategy {
+    /**
+     * If service_ids and trip_ids between active and future feed are all unique, all IDs shall be included
+     * in merged feed. If a service_id from the active calendar has end_date in the future, the end_date shall be
+     * set to one day prior to the earliest start_date in future dataset before appending the calendar record to
+     * the merged file. It shall be ensured that trip_ids between active and future datasets must not match.
+     */
+    DEFAULT,
+    /**
+     * If service_ids and trip_ids in active feed are the same as future feed then the service end date for the
+     * merged feed shall match with future feed’s service end date and the service start date for the merged feed
+     * should be the merged date. All files from the future feed only shall be used in the merged feed.
```

Comment:
It is unclear from this comment what "merged date" is. If a reference to either the active or future feed's dates could be made, that would make it explicit.